server/src/main/resources/buildServerResources/common/buildRetentionEdit.jsp

```diff
@@ -40,7 +40,7 @@
         <props:checkboxProperty name="org.jfrog.artifactory.selectedDeployableServer.buildRetention"
                                 onclick="BS.artifactory.toggleBuildRetentionArgsVisibility()"/>
             <span class="smallNote">
-                Check if you wish to include discard old buils.
+                Check if you wish to discard old builds.
```

Comment:
line: 91
Comma or space-separated list of build numbers to be exclude during the retention procedure.  >> 
Comma or space-separated list of build numbers to be **excluded** during the retention procedure.